openreview/stages/default_content.py

```diff
@@ -464,7 +464,7 @@
         'value': {
             'param': {
                 'type': 'string[]',
-                'regex': '(^$)|[^;,\\n]+(,[^,\\n]+)*'
+                'regex': '.+'
```

Comment:
Could you please add a test reproducing the issue?

Try posting a submission where the value "keywords" is `[""]` and you should expect an exception.

Reply:
Okay, I've added a test for this under `test_venue_matching_setup`.